src/mlabns/mlabns_impl.hpp

```diff
@@ -0,0 +1,114 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
```

Comment:
Now I see why the file here did not appear as a copy with small modifications: because clang-format was run and hence the two files differed too much. As a general rule, I think we SHOULD NOT (capitalized as in RFCs to convey the same meaning) combine refactoring and reformatting because that has the potential of hiding other changes, as I also argued below.